src/zkevm_specs/evm/execution/callop.py

```diff
@@ -171,9 +121,9 @@ def callop(instruction: Instruction):
         is_precompile, FQ(instruction.next.execution_state in precompile_execution_states())
     )
 
-    if is_empty_code_hash == FQ(1) and is_precompile == FQ(0):
+    if call.is_empty_code_hash == FQ(1) and is_precompile == FQ(0):
```

Comment:
Recently the case of \*CALL\* without code execution has been updated due to how non-existing accounts are handled.
Non-existing accounts are now identified by having `code_hash = 0`, which is not the empty hash value.  But those cases should still be handled as no code execution.
So I think it would be better to define:
```python
no_callee_code = call.is_empty_code_hash + call.callee_not_exists
```
And then in this if replace the first check by `no_callee_code == FQ(1)`

I notice that `call.callee_not_exists` is missing from the `CallGadget` in this PR.  I introduced that in a very recent PR in the circuits. See https://github.com/privacy-scaling-explorations/zkevm-circuits/commit/aaaf13ee3020661e048484776bd88686d1659042#diff-08d58ebc59ab6c11bf7c049e859441eed2319a03feb3fdffaddaba73f8a8c829

Reply:
fixed in [a7d329a](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/348/commits/a7d329abe49ca42ed8b9882234e04d853206f6a0)